content/en/docs/content-structure/layout-builder/_index.md

```diff
@@ -177,7 +240,32 @@ A simple 1-level sidebar menu that can display in either the right or left sideb
 | Label | Machine Name               | Type             | Required | Cardinality | Help text | Field Settings | Notes |
 |-------|----------------------------|------------------|----------|-------------|-----------|----------------|-------|
 | Icon  | field_icon                 | Entity reference | no       | 1           |           | Bundle: Image  |
-| Links | field_lb_simple_menu_links | Link             | yes      | unlimited   |
+| Links | field_links | Link             | yes      | unlimited   |
+
+## Staff
+
+- **Machine name:** `lb_staff`
+- **Project:** [lb_staff](<>)
+- **Designs:** [Mobile](<>) | [Desktop](<>)
+
+Displays simple staff member info cards with image, name, title, email
+
+| Label          | Machine Name                     | Type                       | Required | Cardinality | Help text | Field Settings         | Notes |
+|----------------|----------------------------------|----------------------------|----------|-------------|-----------|------------------------|-------|
+| Section heading     | (inherit)     |  |  |  |  |  | H2 |
+| Section subheading  | (inherit)     |  |  |  |  |  | H3 |
+| Staff | field_staff_item | Entity reference revisions | no | unlimited? |  | Bundle: staff_items |
+
+### Staff Item
+
+**Machine name:** `lb_staff_item`
+
+| Label          | Machine Name                     | Type                       | Required | Cardinality | Help text | Field Settings         | Notes |
+|----------------|----------------------------------|----------------------------|----------|-------------|-----------|------------------------|-------|
+| Name | field_item_name | Text (plain) | Yes | 1 |  | Firstname, Lastname | H4 |
+| Image | field_item_image | Entity reference | No | 1 |  |  | If no image uploaded, utilize default |
+| Title | field_item_title | Text (plain) | Yes | 1 | 
```

Comment:
Should this maybe be something different to disambiguate it from `Headings`? Maybe `field_item_job_title`?

Reply:
Yes, I think Job Title works better here, thanks!